Makefile

```diff
@@ -99,6 +100,26 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	stg repair
 	stg import -s ../patch/series
 
+	mkdir $(NON_UP_LOC)
+
+	if [ ! -z ${EXTERNAL_KERNEL_PATCHES} ];  then
+		wget ${EXTERNAL_KERNEL_PATCHES} -O patches.tar.gz
+		tar -zxf patches.tar.gz -C $(NON_UP_LOC)
```

Comment:
Please use `tar -xf` instead of `tar -zxf` here and the one below, in accordance with [this](https://github.com/sonic-net/sonic-buildimage/pull/12428/files#r998395205) comment.